doc/ref_model/chapters/chapter03.md

```diff
@@ -124,24 +124,28 @@ _**Example**: a virtual compute descriptor as defined in TOSCA Simple Profile fo
 <a name="3.2.1.3"></a>
 #### 3.2.1.3 Virtual Storage
 
-A workload can request different types of storage based on data longevity: persistent or ephemeral storage.
+A workload can request storage based on data availability (persistent or ephemeral storage), different types of storage (HDD, SSD, etc.) and storage size.
```

Comment:
Availability is more about the resource being possible.  I think durability could be a better word for length of storage persistance.

Reply:
Sorry, also the types of storage are not typically requested by workload.  Given something like openstack, you would request the storage from a specific pool by name.  While the pool name might be HDD, the actual workload can never know that is really is HDD and is definitely not allowed to know that level of detail about the underlying infrastructure.

Reply:
I agree we could replace **availability** with another term, and I think "retaining policy" may be approximately what we want to say.  IOW, data in ephemeral storage could last a very long time (such as the life of a VM), while data in persistent storage could be overwritten frequently - if its main purpose  is to recover state following a crash or other event. So, associating time with persistent and ephemeral might be right in many circumstances, but not all. 

Also, we already use *type* for  block, object or file storage.  
So, we won't mention HDD or SDD as attributes that workload can request.